doc/source/whatsnew/v0.16.0.rst

```diff
@@ -91,8 +91,7 @@ Interaction with scipy.sparse
 
 Added :meth:`SparseSeries.to_coo` and :meth:`SparseSeries.from_coo` methods (:issue:`8048`) for converting to and from ``scipy.sparse.coo_matrix`` instances (see :ref:`here <sparse.scipysparse>`). For example, given a SparseSeries with MultiIndex we can convert to a `scipy.sparse.coo_matrix` by specifying the row and column labels as index levels:
 
-.. ipython:: python
-   :okwarning:
+.. code-block:: python
```

Comment:
Could be done as a follow up and probably something easy for the community to contribute to, but ideally since we are converting these from `ipython` to `python` directives should add the original output (applicable in a few places below)

Reply:
Certainly wouldn't object, but I think that's fairly low-value since, e.g. the 0.16 version of the docs at https://pandas.pydata.org/pandas-docs/version/0.16.0/whatsnew.html will have the correct rendering.

Reply:
Yea I don’t think you should do here as there’s already enough - just a nice to have a first time contributor could pick up. 

I can open an issue for it later